app/views/components/docs/figure.yml

```diff
@@ -0,0 +1,20 @@
+name: Figure
+description: A responsive image
```

Comment:
There should be some indication of what makes it responsive.
eg A figure containing an image that never exceeds the component’s width